README.md

```diff
@@ -309,7 +189,7 @@ The Feature Labs open source community is happy to provide support to users of C
 4. For everything else, the core developers can be reached by email at [help@featurelabs.com](mailto:help@featurelabs.com).
```

Comment:
Our we still using help@featurelabs.com?

Reply:
nah, i think we can just delete this bullet. they should use the other channels